docker/compose/suzuka-full-node/ledger-should-progress.sh

```diff
@@ -0,0 +1,32 @@
+# install jq if in busybox
+if [ -f /etc/alpine-release ]; then
```

Comment:
Is there a purpose for this standalone script file, when the `ledger-should-progress` inlines a copy of it?